db/Pos/DB/Functions.hs

```diff
@@ -128,6 +141,9 @@ rocksPutBytes k v DB {..} = Rocks.put rocksDB rocksWriteOpts k v
 rocksPutBi :: (Bi v, MonadIO m) => ByteString -> v -> DB -> m ()
 rocksPutBi k v = rocksPutBytes k (encodeStrict v)
 
+rocksPutStore :: (Store v, MonadIO m) => ByteString -> v -> DB -> m ()
```

Comment:
@gromakovsky by the way, can your RocksDB bindings do writes using streaming? I.e. with a lazy ByteString instead of a strict one.